src/apis/group/group.controller.js

```diff
@@ -442,4 +442,34 @@ router.get('/groups/doctors/:doctorId/patients/:patientId', function(req, res) {
     });
 });
 
+
+/**
+ * @swagger
+ * /group/:groupId/groupUserNames:
+ *   get:
+ *     tags:
+ *       - Group
+ *     description: For fetching fullnames of users in group
+ *     produces:
+ *       - application/json
+ *     parameters:
+ *       - name: groupId
+ *         in: path
+ *         description: id of the group
+ *         required: true
+ *         type: integer
+ *         schema:
+ *           $ref: '#/definitions/Group'
+ *     responses:
+ *       200:
+ *         description: list of users in the group
+ */
+
+router.get('/group/:groupId/groupUserNames', function(req, res) {
```

Comment:
URL does not go with the naming convention. You are fetching the users using groupId. Hence it must me /groups/users and pass the groupId as a parameter like groups/users&groupId=groupId or it must be /groups/:groupId/users if you want to use it within the URL. Always keep the objects plural.